.gitignore

```diff
@@ -42,3 +42,5 @@ deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/etc/VERSION
 /.idea
 
 /.vscode
+
+test/integration/testdata/minikube-linux-amd64-latest-stable
```

Comment:
This doesn't seem right. test/ is a code directory and should be immutable: testdata/ doubly so.

Reply:
this is people who run their integration test on desktop and dont wanna accidentally checkin the file.

I thought about downloading the file into a random temp file, but since I am downloading the file in the TestDownloadOnly, and use it in another tests I rather I have a specific location for it.